.travis.yml

```diff
@@ -9,11 +9,6 @@ matrix:
         - ELECTRON_CACHE=$HOME/.cache/electron
         - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
 
-    - os: linux
```

Comment:
don't remove linux build, as we talked - make packing unique for mac os

Reply:
What do you mean by that? Not allow packing for other OS? Some travis configuration?

Reply:
Yes, only Mac OS should do the actual packing, Linux should run build and lint

Reply:
OK